torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -516,6 +528,7 @@ def test_register_rpc_backend_and_set_and_start_rpc_backend(
         )
 
     @requires_process_group_agent("PROCESS_GROUP rpc backend specific test, skip")
+    @_skip_if_tensorpipe_agent
```

Comment:
There is already a decorators that seems to hint that this should only be run for the processgroup agent. Is that decorator not working? Why do we need to add another one?

In principle I think it's fine to introduce a new different decorator (`_skip_if_tensorpipe_agent`) to mark the tests that we should support but currently aren't: it makes it clear that it's a temporary measure rather than the intended situation. However, if this specific test is never supposed to run with anything else than processgroup, then we should not use this new decorator but rather make sure the old one (`requires_process_group_agent`) works.

This comment applies several more times below.